sycl/unittests/scheduler/EnqueueWithDependsOnDeps.cpp

```diff
@@ -227,7 +227,8 @@ TEST_F(DependsOnTests, EnqueueNoMemObjDoubleKernelDepHostBlocked) {
   // kernels on host task completion
   std::vector<EventImplPtr> Events;
 
-  detail::Command *Cmd1 = AddTaskCG(TestCGType::HOST_TASK, Events);
+  detail::Command *Cmd1 =
+      AddTaskCG(TestCGType::HOST_TASK, Events, &CustomHostLambda);
```

Comment:
Do I get it right that the main purpose of CustomHostLambda is to prevent tests from running in parallel?

Reply:
host task should block dependent kernels till its completion, I need to hold host task to check it. I do it with mutex I lock before host task submission and unlock after some intermediate checks. Than I wait till host task completion and check that kernels were enqueued.

Reply:
Thanks!